backend/src/main/java/de/bund/digitalservice/ris/norms/application/port/input/LoadZielnormenUseCase.java

```diff
@@ -7,23 +7,35 @@
 import javax.annotation.Nullable;
 
 /**
- *  Use case for getting information about the Zielnormen that will be created or set to gegenstandslos when applying the {@link ZielnormReference}s of the given Verkündung.
+ *  Use case for getting information about the expressions of Zielnormen that will be created
+ *  or set to gegenstandslos when applying the {@link ZielnormReference}s (containing the geltungszeit)
+ *  to the given Verkündung.
  */
-public interface LoadZielnormenPreviewUseCase {
+public interface LoadZielnormenUseCase {
```

Comment:
I find the word load here a bit confusing now that we plan to use the usecase for both loading the preview and actually **creating** the zielnormen. I think a name like `CreateZielnormenUseCase` would be more fitting. Also the doc comment could use an update :)

Reply:
I agree with the need for renaming, but not 100% sure if a "Create" word is fitting, since it may sound like they are persisted. What about `GenerateZielNormenUseCase`?

Reply:
>  since it may sound like they are persisted
But we do persist them, don't we? Just not in the dry-run case but that should be clear regardless as it's a dry-run

Reply:
Don't really care about the naming, just not load ;)

Reply:
same for me, just wanted to make sure we don't have a misunderstanding about the persisting ^^